.travis.yml

```diff
@@ -21,13 +21,15 @@ cache:
 
 before_install:
   - composer selfupdate
-  - if [ -n "$SYMFONY_VERSION" != "" ]; then
+  - |
```

Comment:
do we really need this?

Reply:
Yes, otherwise it isn't valid YAML and the whole expression needs to be one-lined.

Reply:
it's fine with me. 

but in this case I'm not sure why my version was working fine: https://github.com/sidz/infection/commit/fdfe061defe6d8e71adbc225705a98873db4220e#diff-354f30a63fb0907d4ad57269548329e3R33